scripts/readme/ghactions_driver/workflow_templates/basic_workflow.yml.jinja2

```diff
@@ -2,6 +2,16 @@
 {% block steps %}
 runs-on: ubuntu-latest
 steps:
+  - name: Free Disk Space (Ubuntu)
+    uses: jlumbroso/free-disk-space@main
```

Comment:
seems we are using this GH action: `jlumbroso/free-disk-space`, add some explain to this action (maybe in PR description is enough)